godot-core/src/builtin/quaternion.rs

```diff
@@ -130,8 +134,16 @@ impl Quaternion {
         Quaternion::new(v.x, v.y, v.z, 0.0)
     }
 
+    /// Normalizes the Quaternion.
+    ///
+    /// # Panics
+    /// If the Quaternion has length of 0.
```

Comment:
A bit of a nitpick, but "quaternion" as a noun is lowercase. If you used `Quaternion` as a symbol (don't do it here), it would be uppercase.